### ISSUE TITLE
Project is insufficient

### ISSUE THREAD

**Issue by ukmshi** [christophe-duc/Instappium#2]
Hello!
Thanks you for this project!

It doesn't work properly because of the lack of "appium_actions" when installing from pip. 

Sorry for my poor english.

**Comment by christophe-duc** [comment]
Yes, as stated in the README, more coding is needed, but unfortunately I am out of time to continue this at the moment. Feel free to implement and PR, i will merge your code into the main branch.
